client/signup/steps/domains/index.jsx

```diff
@@ -646,6 +668,10 @@ class DomainsStep extends Component {
 		return this.props.isDomainOnly ? 'domain-first' : 'signup';
 	}
 
+	isTailoredFlow() {
+		return [ 'newsletter', 'link-in-bio' ].includes( this.props.flowName );
```

Comment:
We extracted this repeating check in `@automattic/onboarding` you can import it and use `isNewsletterOrLinkInBioFlow`. We might need to rename it to `isTailoredFlow` at some point.

Reply:
Awesome! will use it, thank you!